splinterd/src/logging/mod.rs

```diff
@@ -0,0 +1,40 @@
+use log::Level;
```

Comment:
This file is missing the copyright header

Reply:
Fixed